docs/user_guide/models/destvi.md

```diff
@@ -230,8 +237,13 @@ impute the spatial pattern of the cell-type-specific gene expression with:
 ### Comparative analysis between samples
 
 To perform differential expression across samples, one can apply a frequentist test by taking samples
-from the parameters of the generative distribution predicted for each spot in question. More details
-can be found in the DestVI paper.
+from the parameters of the generative distribution predicted for each spot in question. 
+### Utilities function
+
+To explore the results of the output of the stLVM, we published a utilities function covering functions
+for automatic thresholding of cell type proportions, a spatial PCA analysis to find main axis of variation
+in spatial gene expression and the described frequentist test for differential expression. Further information
+can be found on [destvi_utils](https://destvi-utils.readthedocs.io/en/latest/installation.html)
 
 [^ref1]: Romain Lopez, Baoguo Li, Hadas Keren-Shaul, Pierre Boyeau, Merav Kedmi, David Pilzer, Adam Jelinski, Eyal David, Allon Wagner, Yoseph Addad, Michael I. Jordan, Ido Amit, Nir Yosef (2021),
```

Comment:
ah, we can change the title here

Reply:
Romain Lopez, Baoguo Li, Hadas Keren-Shaul, Pierre Boyeau, Merav Kedmi, David Pilzer, Adam Jelinski, Ido Yofe, Eyal David, Allon Wagner, Can Ergen, Yoseph Addadi, Ofra Golani, Franca Ronchese, Michael I Jordan, Ido Amit, Nir Yosef. DestVI identifies continuums of cell types in spatial transcriptomics data. Nature Biotechnology (in press), 2022.

Reply:
I made it consistent with other packages and also added link to VAMP prior paper